lib/gel/installer.rb

```diff
@@ -74,10 +75,14 @@ def install_gem(catalogs, name, version)
   end
 
   def load_git_gem(remote, revision, name)
```

Comment:
So here name is either `activesupport`/`activerecord`/etc., which this PR instead groups under `rails`. But far as I know the instruction in the Gemfile would look like `gem "rails", github: "rails/rails"`. Can we carry that name forward to here and use it as the logical grouping?

I have no idea about unforeseen consequences 😄